frontend/apps/marketing/src/components/section/index.tsx

```diff
@@ -0,0 +1,28 @@
+import '@code-dot-org/component-library/cms/section/index.css';
+
+// Import the background image used in the pattern backgrounds on Contentful
+import bgPatternImage from '@public/images/bg-pattern-lines.png';
+
+// Export the Component Definition for use in Contentful Studio
+export {SectionContentfulComponentDefinition} from './sectionContentfulDefinition';
+import DSCOSection, {
+  SectionProps,
+} from '@code-dot-org/component-library/cms/section';
+
+const Section = (props: SectionProps) => {
+  const {background} = props;
+
+  const getBgImage = () => {
```

Comment:
are we planning to support multiple patterns/baground images?

If not - ```
backgroundImageUrl={
      background === 'patternDark' || background === 'patternPrimary'
        ? bgPatternImage.src
        : undefined
    }
```

if yes - it's worth declaring backgroundImageUrl local variable and set value to it by calling `getBgImage` from `useMemo` hook and make `getBgImage` a pure function declared in a file scope (right now it's in component's scope)